examples/example.js

```diff
@@ -109,14 +112,18 @@ async function listTask(owner, token) {
   const metadata = new grpc.Metadata();
   metadata.add("authkey", token);
 
-  const result = await asyncRPC(client, "ListTasks", {
-      smart_wallet_address: process.argv[3]
-  }, metadata);
-
+  const result = await asyncRPC(
+    client,
+    "ListTasks",
+    {
+      smart_wallet_address: process.argv[3],
+    },
+    metadata
+  );
   console.log(`Found ${result.tasks.length} tasks created by`, process.argv[3]);
 
   for (const item of result.tasks) {
-    console.log("\n\ntask id:", item.id, "taskdata", item,"\n=================================\n");
+    console.log(util.inspect(item, { depth: 4, colors: true }));
```

Comment:
Now the response of list looks like this: 
![Google Chrome 2024-11-13 16 44 38](https://github.com/user-attachments/assets/5e1deb45-61f0-46eb-bbf3-21448a1df0fa)